### PR TITLE
Add adapter for Tradermade

### PR DESCRIPTION
**Pull request by boxhock**

### CODE REVIEW

File: tradermade/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@chainlink/tradermade",
```
Comment (krebernisak) [inline tradermade/package.json:2]: Isn't it `@chainlink/tradermade-adapter` now?

**Review by krebernisak — APPROVED** [review]

File: tradermade/package.json (hunk 1 above)
Comment (boxhock) [reply tradermade/package.json:2]: Good point

**Review by krebernisak — APPROVED** [review]

File: tradermade/package.json (hunk 1 above)
Comment (boxhock) [reply tradermade/package.json]: Oh actually, we can't do this until the other PR is merged in. Reverting that commit now

Comment (krebernisak) [reply tradermade/package.json:2]: Ok, let's get that other PR rebased & merged after this then.

**Review by krebernisak — APPROVED** [review]

**Pull request merged**